.pipelines/onebranch/pipeline.image.mirror.dev.yml

```diff
@@ -0,0 +1,64 @@
+#################################################################################
+#                               OneBranch Pipelines                             #
+# This pipeline was created by EasyStart from a sample located at:              #
+#   https://aka.ms/obpipelines/easystart/samples                                #
+# Documentation:  https://aka.ms/obpipelines                                    #
+# Yaml Schema:    https://aka.ms/obpipelines/yaml/schema                        #
+# Retail Tasks:   https://aka.ms/obpipelines/tasks                              #
+# Support:        https://aka.ms/onebranchsup                                   #
+#################################################################################
+
+trigger: none
+pr: none
+
+parameters:
+- name: vsoDeployerBuildID
+  type: string
+  default: latest
+
+variables:
+  CDP_DEFINITION_BUILD_COUNT: $[counter('', 0)] # needed for onebranch.pipeline.version task https://aka.ms/obpipelines/versioning
+  LinuxContainerImage: centos:centos7   # Docker image which is used to build the project https://aka.ms/obpipelines/containers
```

Comment:
I seen ubuntu, debian, now centos. All for RHEL based product. 
@jewzaam this does not look right or sustainable for a long run

Reply:
There are a few things at play here and we can have a longer conversation on the side, but in short we have to work within the Onebranch ecosystem.  We left the mirroring pipeline at the default linux container of centos as it's not generating any binaries, just combining binaries to produce artifacts and manifests for Ev2 to consume.